include/windows/osd.h

```diff
@@ -682,6 +682,8 @@ static inline char* strsep(char **stringp, const char *delim)
 	return ptr;
 }
 
+char *strcasestr(const char *haystack, const char *needle);
```

Comment:
This should already exist on line 872 (please verify) of this file and can be removed.